auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -239,6 +247,10 @@ public Void then(@NonNull Task<GoogleApiClient> task) throws Exception {
         // Facebook sign out
         LoginManager.getInstance().logOut();
 
+        // Twitter sign out
```

Comment:
@samtstern Signing out of Twitter and Facebook should probably a method (something like `signOutProviders`), but that will cause more thrashing in #609. Would you like me to do the refactor anyway and then undo it in #609 or are we not going through with #609?